### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cosama**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,6 @@ pip install becquerel
 
 * Reading additional `Spectrum` file types (N42, CHN, CSV)
 * Writing `Spectrum` objects to various standard formats
-* Fitting spectral features (e.g., gaussian lines with different background models)
```
Comment (markbandstra) [inline README.md:24]: Do we want to note that Poisson likelihood fitting is still in the works?

Comment (jvavrek) [reply README.md:24]: Sounds good

**Review by jvavrek — APPROVED** [review]
Looks good to me

**Review by markbandstra — APPROVED** [review]
Ship it

**Pull request merged**